dev-util/hip/hip-5.1.3-r1.ebuild

```diff
@@ -150,6 +156,11 @@ src_configure() {
 	cmake_src_configure
 }
 
+src_compile() {
+	HIP_PATH=${HIP_S} docs_compile
+	cmake_src_compile
```

Comment:
to actually compile the docs we need to call `docs_compile` somewhere in the compile phase